pytest.ini

```diff
@@ -0,0 +1,5 @@
+[metadata]
+description-file=README.md
+
+[pytest]
+addopts=--verbose --durations=10 --color=yes -s -vv
```

Comment:
Never seen this before, so slowest of 10 repeats. Coloring the output (why is this not default). Looks good. We can automize this with a workflow that test on PR.

Reply:
Added it as an issue #13